library/core/networking/src/testmod/java/org/quiltmc/qsl/networking/test/trackingevents/NetworkingEntityTrackingEventsTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 QuiltMC
```

Comment:
It's 2023!
```suggestion
 * Copyright 2023 QuiltMC
```

Reply:
![meme - 'there i go, still writing 'B.C.' on my checks'](https://user-images.githubusercontent.com/5115825/210862468-275ebf58-c9cf-4c90-8372-b128af97e443.png)